virttest/virsh.py

```diff
@@ -2524,7 +2524,7 @@ def memtune_get(name, key):
     memtune_output = memtune_list(name)
     memtune_value = re.findall(r"%s\s*:\s+(\S+)" % key, str(memtune_output))
     if memtune_value:
-        return int(memtune_value[0])
+        return int(memtune_value[0] if memtune_value[0] != "unlimited" else -1)
```

Comment:
suggest move  memtune_value[0] != "unlimited" to if memtune_value,just something like;
if memtune_value and memtune_value[0] != "unlimited":
          return int(memtune_value[0])
else:
        return -1